docs/src/main/tut/Injection.md

```diff
@@ -47,6 +47,13 @@ implicit val dateToLongInjection = new Injection[java.util.Date, Long] {
 }
 ``` 
 
+We can be less verbose using the `Injection.apply` function:
```

Comment:
I'm not really convinced a definition is more clean this way, I would stick to the full version in the docs...